Source/WebKit/UIProcess/wpe/WebPageProxyWPE.cpp

```diff
@@ -222,6 +222,9 @@ void WebPageProxy::callAfterNextPresentationUpdate(CompletionHandler<void()>&& c
         if (!protectedThis)
             return callback();
 
+        if (!protectedThis->m_drawingArea)
+            return callback();
+
```

Comment:
This is only needed for the old api, the new api doesn't use the drawing area in `callAfterNextPresentationUpdate`, so I think it's better to remove the release assert in `ViewLegacy::callAfterNextPresentationUpdate` and add the early return there.

Reply:
Makes sense. Done.